public/index.html

```diff
@@ -15,6 +15,7 @@
   <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400&amp;display=swap" />
   <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600&amp;display=swap"
     rel="stylesheet">
+  <link href="https://api.fontshare.com/v2/css?f[]=satoshi@700,500,300,400&display=swap" rel="stylesheet">
```

Comment:
This font family is added to the index, meaning it's loaded by the browser each time the application is loaded.

However, this font family is not referenced by any stylesheets. Is it supposed to be in use, or can this be removed?

Reply:
addressed.